src/main/java/com/bookstore/bookstoreapi/data/controller/BookController.java

```diff
@@ -23,4 +25,15 @@ public ResponseEntity<Book> createABook(@RequestBody Book book) {
     }
     return new ResponseEntity<>(createdBook.get(), HttpStatus.CREATED);
   }
+
+  @GetMapping("/books/findbyisbn")
```

Comment:
I am sorry but not the way to do it. Please do it this way:
`@GetMapping("/books/{ISBN}")`